README.md

```diff
@@ -1,3 +1,46 @@
 [![CI](https://github.com/embedded-dev-research/itlab_2023/actions/workflows/ci.yml/badge.svg)](https://github.com/embedded-dev-research/itlab_2023/actions/workflows/ci.yml)
 
 # itlab_2023
+# Name
+## Short description ##
+# **Building a Project**
+To build and run this project locally, follow these steps:
+
+1. **Clone the repository:**  
+   Clone this repository to your local machine using the following command:
+   ```bash
+   git clone https://github.com/embedded-dev-research/itlab_2023.git
```

Comment:
```suggestion
   ```bash
   git clone https://github.com/embedded-dev-research/itlab_2023.git
   ```
```
It is better to close it, because otherwise syntax highlighting breaks sometimes and IIRK it could work not everywhere

Reply:
Do I need to remove this line? I didn't really understand what to fix

Reply:
suggestion is buggy. I meant this:
   \`\`\`bash
   git clone https://github.com/embedded-dev-research/itlab_2023.git
   \`\`\` <- add this line